code/modules/power/smes.dm

```diff
@@ -148,68 +152,70 @@
 			to_chat(user, "<span class='warning'>You must not be on the same tile as [src].</span>")
 			return
 
-		//Direction the terminal will face to
-		var/tempDir = get_dir(user, src)
-		switch(tempDir)
+		// Direction the terminal will face to
+		var/temporary_direction = get_dir(user, src)
+		switch(temporary_direction)
 			if(NORTHEAST, SOUTHEAST)
-				tempDir = EAST
+				temporary_direction = EAST
 			if(NORTHWEST, SOUTHWEST)
-				tempDir = WEST
-		var/turf/tempLoc = get_step(src, REVERSE_DIR(tempDir))
-		if(isspaceturf(tempLoc))
+				temporary_direction = WEST
```

Comment:
... why do we disable diagonal clicking in line 130 but have a failsave here for diagonals?

Reply:
Discussed on Discord!